tests/ci/cdk/cdk/aws_lc_github_ci_x509_stack.py

```diff
@@ -0,0 +1,93 @@
+from aws_cdk import Duration, Stack, aws_codebuild as codebuild, aws_s3 as s3
+from constructs import Construct
+from util.build_spec_loader import BuildSpecLoader
+from util.metadata import (
+    GITHUB_PUSH_CI_BRANCH_TARGETS,
+    GITHUB_REPO_NAME,
+    GITHUB_REPO_OWNER,
+)
+
+
+class AwsLcGitHubX509CIStack(Stack):
+    def __init__(
+        self,
+        scope: Construct,
+        id: str,
+        **kwargs,
+    ) -> None:
+        super().__init__(scope, id, **kwargs)
+
+        git_hub_source = codebuild.Source.git_hub(
+            owner=GITHUB_REPO_OWNER,
+            repo=GITHUB_REPO_NAME,
+            webhook=True,
+            webhook_filters=[
+                codebuild.FilterGroup.in_event_of(
+                    codebuild.EventAction.PULL_REQUEST_CREATED,
+                    codebuild.EventAction.PULL_REQUEST_UPDATED,
+                    codebuild.EventAction.PULL_REQUEST_REOPENED,
+                ),
+                codebuild.FilterGroup.in_event_of(
+                    codebuild.EventAction.PUSH
+                ).and_branch_is(GITHUB_PUSH_CI_BRANCH_TARGETS),
+            ],
+            webhook_triggers_batch_build=True,
+        )
+
+        self.reports_bucket = s3.Bucket(
```

Comment:
Not a blocker, just genuinely curious.
Is there a difference between saving these in the cdk.App class V.S. defining these as [local variables](https://github.com/aws/aws-lc/blob/8e46915dbcd56c3c65de157d1acd9570b57df4b2/tests/ci/cdk/cdk/aws_lc_github_ci_stack.py#L39-L41) like what we were originally doing?

Reply:
For this particular instance no, but it can be useful when you start having inter-stack dependencies where you want to reference let's say the reporting bucket ARN etc. I think when I did this I had my AWS Pipeline PoC in mind where I was leveraging some of that.